lib/command.ts

```diff
@@ -54,10 +55,19 @@ export function command(
     const middlewares = commandMiddlewareFactory(commandMiddleware);
     handler = handler || (() => {});
 
-    if (Array.isArray(cmd)) {
-      aliases = cmd.slice(1);
+    // If an array is provided that is all CommandHandlerDefinitions, add
+    // each handler individually:
+    if (Array.isArray(cmd) && !cmd.find(c => typeof c === 'string')) {
+      for (const command of cmd) {
+        self.addHandler(command);
+      }
+    } else if (Array.isArray(cmd)) {
+      // Assume the array is a cmd, followed by aliases, if the array contains
+      // any strings [cmd, 'alias1', 'alias2']:
+      aliases = cmd.slice(1) as string[];
       cmd = cmd[0];
     } else if (isCommandHandlerDefinition(cmd)) {
+      // An object was provided rather than individual parameters:
```

Comment:
i suspect the types may narrow better with this:
```suggestion
    if (Array.isArray(cmd)) {
      if (cmd.every(c => typeof c === 'string')) {
        ([cmd, aliases] = cmd);
      } else {
        for (const command of cmd) {
          self.addHandler(command);
        }
      }
    } else if (isCommandHandlerDefinition(cmd)) {
```